src/shorthands/JLine.jl

```diff
@@ -0,0 +1,22 @@
+function _JLine(p1, p2, color)
+    sethue(color)
+    line(p1, p2, :stroke)
+    return p2
+end
+
+"""
+    JLine(p1::Point, p2::Point, color="black")
+
+Make a line between two points, pt1 and pt2 and do an action.
+Returns the final point of the line
+"""
+JLine(p1::Point, p2::Point; color = "black") =
+    (args...; color = color, p1 = p1, p2 = p2) -> _JLine(p1, p2, color)
+
+"""
+    JLine(pt::Point)
```

Comment:
the color is missing in this docstring. I would also like to have `linewidth` for `setline` default should be 1.